praetorian_cli/ui/conversation/textual_chat.py

```
@@ -0,0 +1,622 @@
+                    if response.get('error'):
+                        self.add_system_message(f"Error: {response.get('error')}")
+                        self.update_status("Error - Ready for next message")
+                        return
+                    
+                    # Update status and wait for AI response
+                    self.update_status("Waiting for AI response...")
+                    
+                    # Poll for AI response
+                    await self.wait_for_ai_response()
+                    
+                except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)